### ISSUE TITLE
Add possibility to inherit global navigation from parent web

### ISSUE THREAD

**Issue by olemp** [olemp/js-sites-core#13]
Add possibility to inherit global navigation from parent web. Preferably done in object handler WebSettings.

**Comment by olemp** [comment]
_Initial thought:_

LocalNavigation

``` json
     "InheritGlobalNavigationFromParent": false,
     "GlobalNavigation": [],
     "LocalNavigation": []
```

| Attibute | Type | Description |
| --- | --- | --- |
| InheritGlobalNavigationFromParent | boolean | The InheritGlobalNavigationFromParent entries of the Navigation, optional collection of elements |
| GlobalNavigation | Array<[NavigationNode](#navigationnode)> | The GlobalNavigation entries of the Navigation, optional collection of elements |
| LocalNavigation | Array<[NavigationNode](#navigationnode)> | The LocalNavigation entries of the Navigation, optional collection of elements |

But do we really ever need to set global navigation nodes per subsite? Don't think so.

**Comment by tarjeieo** [comment]
I actually used _QuickLaunch_ and _UseShared_, as that's what is being used by the SP.js Navigation class. Open for changing it of course

**Comment by olemp** [comment]
That's even better.

Released https://github.com/Puzzlepart/js-sites-core/releases/tag/v1.8.1

Closing after confirmed working!

**Comment by olemp** [comment]
The changes I did was to change the rejects to resolve. I don't want the setup to stop. So all ProvisionObjects functions in all object handlers should resolve. Should still send error and warning messages.

Had to move ConfigureQuickLaunch out from the class to the namespace because it failed. Still haven't found a way to call functions from the same class. It gives a weird error. Might have to do with the way we queue up the handlers.

Closing. Nice first contribution @tarjeieo !
